subprojects/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc

```diff
@@ -0,0 +1,336 @@
+        │       └── demo
+        │           └── App.java        // <5>
+        └── test
+            └── java
+                └── demo
+                    └── AppTest.java    // <6>
+----
+<1> Generated folder for wrapper files
+<2> Gradle wrapper start scripts
+<3> Settings file to define build name and subprojects
+<4> Build script for `app` dependencies
+<5> Default Java source folder for `app`
+<6> Default Java test source folder for `app`
+
+The `authoring-tutorial` folder is the *root project directory*.
+Inside the root project directory are one or more subprojects, build scripts, and the Gradle wrapper.
+
+While the Gradle Wrapper is local to the root project, the Gradle executable is found in the `GRADLE_USER_HOME`.
+
+The `GRADLE_USER_HOME`, which defaults to `USER_HOME/.gradle`, is also where Gradle stores its global configuration properties, initialization scripts, caches, log files and more.
+
+== Step 3. Review the Gradle Files
+
+The `settings.gradle.kts` file has two interesting lines:
+
+.settings.gradle.kts
+[source, kotlin]
+----
+rootProject.name = "authoring-tutorial"
+include("app")
+----
+
+- `rootProject.name` assigns a name to the build, which overrides the default behavior of naming the build after the directory it’s in.
+- `include("app")` defines that the build consists of one subproject called `app` that contains its own source code and build logic.
+
+More subprojects can be added by additional `include()` statements.
+
+Our build contains one subproject called `app` that represents the Java application we are building.
+It is configured in the `app/build.gradle(.kts)` file:
+
+.build.gradle.kts
+[source, kotlin]
+----
+plugins {
+    id("application")                                               // <1>
+}
+
+repositories {
+    mavenCentral()                                                  // <2>
+}
+
+dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter:5.9.3")     // <3>
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+    implementation("com.google.guava:guava:32.1.1-jre")             // <4>
+}
+
+java {                                                              // <5>
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(19))
+    }
+}
+
+application {
+    mainClass.set("authoring.tutorial.App")                         // <6>
+}
+
+tasks.named<Test>("test") {
+    useJUnitPlatform()                                              // <7>
+}
+----
+<1> Apply the application plugin to add support for building a CLI application in Java.
+<2> Use Maven Central for resolving dependencies.
+<3> Use JUnit Jupiter for testing.
+<4> This dependency is used by the application.
+<5> Define the toolchain version.
+<6> Define the main class for the application.
+<7> Use JUnit Platform for unit tests.
+
+The build script in the `app` subproject directory declares the dependencies that the `app` code will need to be assembled, tested, and published.
```

Comment:
```suggestion
The build script in the `app` subproject directory declares the dependencies that the `app` code will need to be assembled and tested.
```

We shouldn't talk about publishing at all at this point, the setup is much more involved.